ci-operator/step-registry/baremetalds/setup/baremetalds-setup-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: baremetalds-setup
```

Comment:
With the steps-based approach, should these have more generic names? So it's usable by someone to do some other platform on packet?

Reply:
The idea here was to break in two (one for packet and one for dev-scripts) once we had a consolidate running version